admin/src/App.tsx

```diff
@@ -13,8 +21,12 @@ import { CustomAppBar } from './appbar/CustomAppBar';
 import englishMessages from 'ra-language-english';
 import dutchMessages from 'ra-language-dutch';
 import polyglotI18nProvider from 'ra-i18n-polyglot';
+import { CardUpload } from './create/CardUpload';
+import Card from '@mui/material/Card';
+import CardContent from '@mui/material/CardContent';
+import { Button, Typography } from '@mui/material';
 
-const dataProvider = CustomDataProvider('http://localhost:3002/api');
+const dataProvider = CustomDataProvider('http://localhost:3003/api');
```

Comment:
Will this work correctly? Maybe try https://stackoverflow.com/questions/43813415/webpack-dev-server-proxy-doesnt-redirect-to-api-backend for development, and set this to '/api' which will then work correctly in production.